ClosedXML/Excel/Cells/IXLCell.cs

```diff
@@ -310,8 +310,31 @@ public interface IXLCell
 
         IXLCell CopyTo(String target);
 
+        /// <summary>
```

Comment:
Ok, so for this interface I have some other ideas. 

Firstly, for a user, I think it is going to be confusing to have both `ValueCached` and `ValueCalculated`. I think we should have only 1 and I think ideally it should return `Object`. That will be a breaking change, so maybe we should phase it in with keeping both for now, and marking one as obsolete. We'll need a property name that correctly describes both. Maybe `Object CachedValue {get;}` is OK.